doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -82,6 +82,36 @@ To coerce an object :g:`t:C t₁..tₙ` of ``C`` towards ``D``, we have to
 apply the coercion ``f`` to it; the obtained term :g:`f _.._ t` is
 then an object of ``D``.
 
+Reverse Coercions
+-----------------
+
+A term is reverse-coerced only if it cannot be coerced (directly) to its expect type.
+A coercion is said to be reversible if it is declared with with the :attr:`reverse` attribute.
```

Comment:
Can you reword to begin with "Reverse coercions are/allow/..." - a definition?
The second sentence is not very helpful as written--almost a tautology.